src/V3Task.cpp

```diff
@@ -462,7 +462,7 @@ class TaskVisitor final : public VNVisitor {
             UINFO(9, "     Port " << portp << endl);
             UINFO(9, "      pin " << pinp << endl);
             if (inlineTask) {
-                pinp->unlinkFrBack();  // Relinked to assignment below
+                pushDeletep(pinp->unlinkFrBack());  // Cloned in assignment below
```

Comment:
Probably need VL_DO_DANGLING on the three deletes in this file (or comment otherwise).

Reply:
They are all used later, but the control flow is a mess, so I disn't want to do

if (a) foo->replaceWith(bar)
dosomethingwith(foo)
if (a) pushDeleteo(foo)

Added comments